### PR TITLE
v4.0.0

### PR DESCRIPTION
**Pull request by abrahamcalsin**
- [x] add chakra ui
- [x] add notion api

### CODE REVIEW

File: pages/404.tsx [hunk 1]
```diff
@@ -7,19 +8,44 @@ const custom404 = () => {
...
+            fontSize="md"
+            mt="18px"
+            py="12px"
+            px="18px"
+            borderRadius="full"
+            transitionProperty="opacity"
+            _hover={{
+              opacity: 0.8,
+            }}
+            className="shadow-lg shadow-primary/5"
```
Comment (yonycalsin) [inline pages/404.tsx:43]: do you stilll have tailwindcss ?

File: pages/projects/index.tsx [hunk 2]
```diff
@@ -1,31 +1,47 @@
...
+      notFound: true,
+    }
+  }
+
+  return {
+    props: {
+      projects: response.results,
+    },
+
+    revalidate: 10,
```
Comment (yonycalsin) [inline pages/projects/index.tsx:43]:
```suggestion
    },
    revalidate: 10,
```

File: src/components/article-body/index.tsx [hunk 3]
```diff
@@ -19,30 +20,61 @@ export function ArticleBody(props: ArticleBodyProps) {
...
-        <hr className="my-4.5 border-primary-800/10" />
-        <div className="flex gap-x-px text-primary-600">
-          <ShareArticle articleLink={articleSlug} articleTitle={shareTitle} />
-          <span className="mx-1">•</span>
-          <EditArticle slug={articleSlug} />
-        </div>
-      </div>
-    </>
+    <Box my={{ base: '0', sm: '76px' }}>
+      <Box>
```
Comment (yonycalsin) [inline src/components/article-body/index.tsx:24]: is not necessary the box componente, becouse you are not applying some styles or something like that

```suggestion
      <div>
```

File: src/components/article-body/index.tsx [hunk 4]
```diff
@@ -19,30 +20,61 @@ export function ArticleBody(props: ArticleBodyProps) {
...
+          justifyContent="space-between"
+          textColor="primary.600"
+          fontSize={{ base: 'sm', sm: 'md' }}
+          mt={{ base: '16px', sm: '28px' }}
+          mb="16px"
+          _dark={{
+            textColor: 'primary.400',
+          }}
+        >
+          <Box>
```
Comment (yonycalsin) [inline src/components/article-body/index.tsx:46]: you don't need to use the box component

```suggestion
          <div>
```

File: src/components/date-format-article/index.tsx [hunk 5]
```diff
@@ -12,5 +12,5 @@ export function DateFormatArticle(props: DateFormatArticleProps) {
 
   const date = dayjs(dateArticle).locale(en).format(dateFormat.article)
 
-  return <span className="font-regular text-inherit dark:text-inherit capitalize">{date}</span>
+  return <span className="font-regular text-inherit capitalize">{date}</span>
```
Comment (yonycalsin) [inline src/components/date-format-article/index.tsx:15]: I think you can use the text component here instead

File: src/components/navbar-mobile/index.tsx [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
```
Comment (yonycalsin) [inline src/components/navbar-mobile/index.tsx:1]:
```suggestion
import * as React from 'react'
```

File: src/components/navbar-mobile/index.tsx [hunk 7]
```diff
@@ -37,31 +50,48 @@ function NavigationBottom(props: NavigationBottomProps) {
...
 
   return (
-    <Link href={href}>
+    <RouterLink href={href} passHref>
       <a
         className={clsx(
           'flex justify-center items-center h-full w-7.7 text-xs border-t-4 border-primary-800 dark:border-primary-200 flex-col fill-current',
           !isActiveSelect &&
-            'text-primary-500 dark:text-primary-600 border-t-primary/0 dark:border-t-white/0 fill-transparent',
+            `text-primary-500 dark:text-primary-600 border-t-primary/0 dark:border-t-white/0 fill-transparent`,
```
Comment (yonycalsin) [inline src/components/navbar-mobile/index.tsx:58]: you don't need to use template-strings

```suggestion
            'text-primary-500 dark:text-primary-600 border-t-primary/0 dark:border-t-white/0 fill-transparent',
```

File: src/components/pill-technologies/index.tsx [hunk 8]
```diff
@@ -17,25 +18,42 @@ function Technology(props: TechnologyProps) {
...
-    <a
+    <Link
       href={href}
-      target="_blank"
-      className="inline-flex w-9.5 h-4.5 sm:w-11.2 sm:h-5.2 px-1 sm:px-1.5 text-primary-800 dark:text-primary-200 bg-primary-100 dark:bg-secondary/10 rounded-md border-px border-primary-600 dark:border-secondary transition-opacity hover:opacity-80"
       rel="noreferrer"
+      display="inline-flex"
+      bg="primary.100"
+      textColor="primary.800"
+      width={{ base: '76px', sm: '90px' }}
```
Comment (yonycalsin) [inline src/components/pill-technologies/index.tsx:27]: I would recommend you, to use the spacing defined by chakra-ui

the spacings

https://chakra-ui.com/docs/styled-system/theme#spacing

the spacings are rem format, so you will need to conver to px just as a reference
https://nekocalc.com/px-to-rem-converter

File: src/screens/home/home.tsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+import { Meta } from '~/components/meta'
+import { MainLayout } from '~/layouts'
+
+import { Content } from './components/content'
+
+export function HomeScreen() {
+  return (
+    <MainLayout>
+      <Meta />
```
Comment (yonycalsin) [inline src/screens/home/home.tsx:11]: this componente should go inside the page file, pages/index.tsx

**Review by yonycalsin — CHANGES_REQUESTED** [review]

File: pages/404.tsx (hunk 1 above)
Comment (abrahamcalsin) [reply pages/404.tsx:43]: yes, I have not yet migrated everything to chakra-ui

File: src/components/article-body/index.tsx (hunk 3 above)
Comment (abrahamcalsin) [reply src/components/article-body/index.tsx:24]: thanks 🎊

File: src/components/date-format-article/index.tsx (hunk 5 above)
Comment (abrahamcalsin) [reply src/components/date-format-article/index.tsx:15]: 🎊

File: src/components/pill-technologies/index.tsx (hunk 8 above)
Comment (abrahamcalsin) [reply src/components/pill-technologies/index.tsx:27]: thanks 🎊🎊

File: src/screens/home/home.tsx (hunk 9 above)
Comment (abrahamcalsin) [reply src/screens/home/home.tsx:11]: I do not understand.
I'm importing that component into the page file, pages/index.tsx 🤔

Comment (yonycalsin) [reply src/screens/home/home.tsx:11]: I mean to move the declaracion of meta outside of the screen component

something like this

/pages/index.tsx
```tsx
const HomePage = ()=> {
   return (
      <>
         <Meta/>
         <HomeScreen/>
      </>
   )
}
```

Comment (abrahamcalsin) [reply src/screens/home/home.tsx:11]: ok 🎊

Comment (yonycalsin) [reply src/screens/home/home.tsx:11]: all pages

Comment (abrahamcalsin) [reply src/screens/home/home.tsx:11]: ok, thanks

**Review by yonycalsin — APPROVED** [review]
🎉

**Pull request merged**